README.md

```diff
@@ -28,21 +28,22 @@ New here or looking to install? Check [our website](https://randovania.github.io
 
 <!-- Begin WELCOME -->
 
-Here you will be able to randomize many aspects of either game, while still being ensured it's possible to
-finish without any trick or glitch! What can be randomized?
+Randovania can randomize many aspects of its supported games, all while still ensuring they're completable 
+without any unintended tricks or glitches. Its features include:
 
-* Randomize what can be found in each pickup location, including major upgrades, expansions, keys and artifacts.
+* Randomizing what can be found in each pickup location. Weapons, keys, and more can end up in
+  completely new places.
 
-* Play with multiple people, via multiworld sessions. Your pickups will be shuffled among the games of
-everyone involved, no matter what game they're playing!
+* Use Multiworld sessions to shuffle items between multiple separate games, alone or with friends.
+  All Multiworld games are compatible with each other - mix and match as you like!
 
-* Randomize where each teleporter goes, or what you need to unlock a translator gate. In either case,
-there's advanced options for how they're shuffled.
+* Randomize how areas connect to each other, or what resources are required to travel between areas.
+  These options are highly customizable, letting you limit or unleash the chaos.
```

Comment:
the chaos or just chaos?

Reply:
I think "the chaos" reads best with the parallelism of "limit the chaos" and "unleash the chaos" vs. "limit chaos" and "unleash chaos." The former sounds more fun & lighthearted, which was the aim.

Reply:
Might be a non-native thing then. "limit the chaos" sounds weird to me personally.